services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationControllerITest.kt

```diff
@@ -52,6 +59,7 @@ private const val BASE_URL = "/hakemukset"
 @WebMvcTest(ApplicationController::class)
 @Import(IntegrationTestConfiguration::class)
 @ActiveProfiles("test")
+@WithMockUser(USERNAME)
 class ApplicationControllerITest(@Autowired override val mockMvc: MockMvc) : ControllerTest {
```

Comment:
Are there any functional changes in this file or is it all just refactoring? I can't find any.

Reply:
Delete checks that deleteWithOrphanGeneratedHankeRemoval is called (previously called delete).